.circleci/config.yml

```diff
@@ -9,6 +9,14 @@ workflows:
           filters:
             branches:
               ignore: languages
+      - rips-test:
+          context: org-global
```

Comment:
Is the global context necessary for this workflow? What variables are required?

Reply:
I defined three variables on the CircleCI job for the `planet4-master-theme`, but I'm not sure if that's the "global" context or not, the variables are: `RIPS_BASE_URI`, `RIPS_USERNAME` and `RIPS_PASSWORD`, which are required for `rips-cli` to authenticate.

![image](https://user-images.githubusercontent.com/340766/56941376-85ba9580-6aea-11e9-959f-74e1e3066bc9.png)

Will the variables still be available if we remove this line?

Reply:
I think a new context specifically for RIPS test jobs might be the way to go - https://circleci.com/gh/organizations/greenpeace/settings#contexts

Adding these vars to a new context and assign that instead of org-global will reduce the number of secrets we're unnecessarily exposing to environments where they are not required.

Reply:
Using the new `rips-scans` context.